src/main/java/io/github/wimdeblauwe/errorhandlingspringbootstarter/handler/GqlConstraintViolationExceptionHandler.java

```diff
@@ -0,0 +1,145 @@
+package io.github.wimdeblauwe.errorhandlingspringbootstarter.handler;
+
+import graphql.GraphQLError;
+import graphql.GraphqlErrorBuilder;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.*;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.ErrorCodeMapper;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.ErrorMessageMapper;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.GraphQlErrorMapper;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.HttpStatusMapper;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.ConstraintViolationException;
+import jakarta.validation.ElementKind;
+import jakarta.validation.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class GqlConstraintViolationExceptionHandler extends AbstractGraphqlExceptionHandler {
```

Comment:
Lots of duplication with `ConstraintViolationApiExceptionHandler`. We should find a way to avoid that.

Reply:
Yes here it is a fact, indeed. How would you propose to tackle this? We have multiple options:

1. Static Utils class to handle ConstraintViolationException -> ApiErrorResponse
2. Bean helper class that translates ConstraintViolationException -> ApiErrorResponse (this increases extensibility)
3. Specialized method (+ it's dependencies) in the super class to handle this specific behavior